app/config/parameters.yml.dist

```diff
@@ -73,3 +73,6 @@ parameters:
 
     # pocket import
     pocket_consumer_key: xxxxxxxx
+    
+    # sites credentials
+    sites_credentials: {}
```

Comment:
I see. Well, let me know. I agree that this way to store credentials is far from ideal.